internal/paascharm/gostructs.go

```diff
@@ -0,0 +1,140 @@
+package paascharm
+
+import (
+	"fmt"
+	"log"
+	"strings"
+	"unicode"
+)
+
+// Map from integration key in charmcraft.yaml to Go name
+var IntegrationsToGoName = map[string]string{
+	"mongodb":    "MongoDB",
+	"mysql":      "MySQL",
+	"postgresql": "PostgreSQL",
+	"redis":      "Redis",
+	"s3":         "S3",
+	"saml":       "SAML",
+}
+
+// Map from database integration key in charmcraft.yaml to its prefix in env vars
+var DatabaseIntegrationNameToPrefix = map[string]string{
+	"mongodb":    "APP_MONGODB_",
+	"mysql":      "APP_MYSQL_",
+	"postgresql": "APP_POSTGRESQL_",
+	"redis":      "APP_REDIS_",
```

Comment:
Couldn't you reuse the `ConfigOptionsPrefix` constant ?

Reply:
Refactored to only have the constant "APP_" in one place